docs/tutorials/connecting-to-ev3dev-with-ssh.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Connecting to Ev3dev Using SSH
```

Comment:
I think that's a stylistic choice for the logo instead of a grammatical one. Personally, the only potential options that I think are explicitly _incorrect_ are EV3DEV and EV3dev.